biz.aQute.bndlib/src/aQute/bnd/osgi/Builder.java

```diff
@@ -1458,6 +1459,17 @@ public boolean doNotCopy(String v) {
 		return getDoNotCopy().matcher(v).matches();
 	}
 
+	public boolean doNotCopy(File from) {
+		if (doNotCopy(from.getName())) {
+			return true;
+		}
+
+		URI baseURI = getBase().toURI();
```

Comment:
@pkriens suggested to put this in an instance var so you are not creating the baseURI many times. You could even add `getBaseURI()` to Processor (next to `getBase()`) to push the function to the superclass since others can then use.